client/GameComponents/PlayerRow.jsx

```diff
@@ -70,68 +35,6 @@ class PlayerRow extends React.Component {
         }
     }
 
-    onDiscardedCardClick(event, cardId) {
-        event.preventDefault();
-        event.stopPropagation();
-
-        if(this.props.onDiscardedCardClick) {
-            this.props.onDiscardedCardClick(cardId);
-        }
-    }
-
-    getHand(needsSquish) {
-        var cardIndex = 0;
-        var handLength = this.props.hand ? this.props.hand.length : 0;
-        var requiredWidth = handLength * 64;
-        var overflow = requiredWidth - 342;
-        var offset = overflow / (handLength - 1);
-
-        var hand = _.map(this.props.hand, card => {
-            var left = (64 - offset) * cardIndex++;
-
-            var style = {};
-            if(needsSquish) {
-                style = {
-                    left: left + 'px'
-                };
-            }
-
-            return (<Card key={ card.uuid } card={ card } style={ style } disableMouseOver={ !this.props.isMe } source='hand'
-                onMouseOver={ this.props.onMouseOver }
-                onMouseOut={ this.props.onMouseOut }
-                onClick={ this.props.onCardClick }
-                onDragDrop={ this.props.onDragDrop } />);
-        });
-
-        return hand;
-    }
-
-    getDrawDeck() {
```

Comment:
:+1: this was supposed to go away when i converted it to a CardCollection (which now I think of it is a really dumb name)